go.mod

```diff
@@ -18,7 +20,7 @@ require (
 	github.com/agext/levenshtein v1.2.2 // indirect
 	github.com/apparentlymart/go-textseg v1.0.0 // indirect
 	github.com/apparentlymart/go-textseg/v13 v13.0.0 // indirect
-	github.com/aws/aws-sdk-go v1.36.1 // indirect
+	github.com/aws/aws-sdk-go-v2 v1.9.0
```

Comment:
This should be moved in the first require then

Reply:
ah right, will fix

Reply:
I did a rebase and fixed the above, I can do another rebase once https://github.com/cyrilgdn/terraform-provider-postgresql/pull/146 is merged @cyrilgdn and we should be all set!